### PR TITLE
Emit metrics to indicate the test success and execution time

### PR DESCRIPTION
**Pull request by bhatpmk**
# Description

This change adds utility to emit metrics to indicate the test success and execution time, from the CI pipeline. The Jenkins scripts for Kind AT, multi-cluster AT, uninstall, OCI DNS and upgrade jobs are updated right now, remaining jobs will be updated after observing the behavior for couple of runs.

# Checklist 

As the author of this PR, I have:

- [ ] Checked that I included or updated copyright and license notices in all files that I altered
- [ ] Added or updated unit tests for any new functions I added
- [ ] Added or updated integration tests if appropriate
- [ ] Added or updated acceptance tests if appropriate

Code reviewer, please confirm this PR:

- [ ] Addressed the requirement and meets the acceptance criteria
- [ ] Does not introduce unrelated or spurious changes
- [ ] Does not introduce any unapproved dependency
- [ ] Makes sense and it easy to understand, and/or difficult areas of code are clearly documented so that they can be understood

### CODE REVIEW

File: ci/uninstall/Jenkinsfile [hunk 1]
```diff
@@ -99,6 +99,11 @@ pipeline {
         INSTALL_CONFIG_FILE = "${GO_REPO_PATH}/verrazzano/platform-operator/config/samples/install-default.yaml"
         WEBLOGIC_PSW = credentials('weblogic-example-domain-password')
         DATABASE_PSW = credentials('todo-mysql-password')
+
+        // used to emit metrics
+        SAURON_CRED = credentials('verrazzano-sauron')
+        PROMETHEUS_GW_URL = credentials('v8o-dev-sauron-prometheus-url')
+        TEST_ENV_LABEL = "uninstall"
```
Comment (shihchang) [inline ci/uninstall/Jenkinsfile:106]: The testEnv value should be 'ocidns_oke_' + "${params.OKE_CLUSTER_VERSION}"

File: ci/oke-ocidns/Jenkinsfile [hunk 2]
```diff
@@ -146,6 +146,11 @@ pipeline {
         OCI_OS_NAMESPACE = credentials('oci-os-namespace')
         OCI_OS_ARTIFACT_BUCKET="build-failure-artifacts"
         OCI_OS_BUCKET="verrazzano-builds"
+
+        // used to emit metrics
+        SAURON_CRED = credentials('verrazzano-sauron')
+        PROMETHEUS_GW_URL = credentials('v8o-dev-sauron-prometheus-url')
+        TEST_ENV_LABEL = "oke_oci_dns"
```
Comment (shihchang) [inline ci/oke-ocidns/Jenkinsfile:153]: should be 'ocidns_oke_' + "${params.OKE_CLUSTER_VERSION}" unless we have another label for k8s-version

File: ci/multicluster/Jenkinsfile [hunk 3]
```diff
@@ -155,6 +155,11 @@ pipeline {
         OCI_OS_NAMESPACE = credentials('oci-os-namespace')
         OCI_OS_ARTIFACT_BUCKET="build-failure-artifacts"
         OCI_OS_BUCKET="verrazzano-builds"
+
+        // used to emit metrics
+        SAURON_CRED = credentials('verrazzano-sauron')
+        PROMETHEUS_GW_URL = credentials('v8o-dev-sauron-prometheus-url')
+        TEST_ENV_LABEL = "multi_cluster_${params.TEST_ENV}_${params.TEST_ENV == 'kind' ? params.KIND_CLUSTER_VERSION : params.OKE_CLUSTER_VERSION}"
```
Comment (shihchang) [inline ci/multicluster/Jenkinsfile:162]: "${params.TEST_ENV}" + '_' + "${params.TEST_ENV == 'kind' ? params.KIND_CLUSTER_VERSION : params.OKE_CLUSTER_VERSION }"

File: ci/uninstall/Jenkinsfile (hunk 1 above)
Comment (bhatpmk) [reply ci/uninstall/Jenkinsfile]: Done

File: ci/oke-ocidns/Jenkinsfile (hunk 2 above)
Comment (bhatpmk) [reply ci/oke-ocidns/Jenkinsfile]: Done

File: ci/multicluster/Jenkinsfile (hunk 3 above)
Comment (bhatpmk) [reply ci/multicluster/Jenkinsfile]: Done

**Review by shihchang — APPROVED** [review]

**Pull request merged**